webui/react/src/shared/styles/antd.scss

```diff
@@ -34,8 +34,8 @@ html {
     border-color: var(--theme-ix-border-active);
     color: var(--theme-ix-on-active);
   }
-  .ant-btn[disabled]:not(.ant-btn-link):not(.ant-btn-primary),
-  .ant-btn[disabled]:not(.ant-btn-link):not(.ant-btn-primary):hover {
+  .ant-btn[disabled]:not(.ant-btn-link, .ant-btn-primary),
```

Comment:
This seems like a semantic change.

Reply:
let me check

Reply:
I changed a rule https://stylelint.io/user-guide/rules/list/selector-not-notation/ to have backward compatibility, so reverted these changes